File: seed/java-sdk/exhaustive/enable-public-constructors/src/main/java/com/snippets/Example38.java
package com.snippets;

import com.seed.exhaustive.SeedExhaustiveClient;
import com.seed.exhaustive.resources.types.union.types.Animal;
import com.seed.exhaustive.resources.types.union.types.Dog;

public class Example38 {
    public static void run() {
        SeedExhaustiveClient client = SeedExhaustiveClient
            .builder()
            .token("<token>")
            .url("https://api.fern.com")
            .build();

        client.endpoints().union().getAndReturnUnion(
            Animal.dog(
                Dog
                    .builder()
                    .name("name")
                    .likesToWoof(true)
                    .build()
            )
        );
    }
}

File: seed/csharp-sdk/exhaustive/redact-response-body-on-error/src/SeedApi.DynamicSnippets/Example38.cs
using SeedExhaustive;

public partial class Examples
{
    public static async Task Example38()
    {
        var client = new SeedExhaustiveClient(
            token: "<token>",
            clientOptions: new ClientOptions {
                BaseUrl = "https://api.fern.com"
            }
        );

        await client.Endpoints.Params.GetWithPathAndErrorsAsync(
            "param"
        );
    }

}
